maestro/trainer/models/florence_2/training.py

```diff
@@ -109,15 +118,15 @@ def train(configuration: TrainingConfiguration) -> None:
     processor, model = load_model(
         model_id_or_path=best_model_path,
     )
-    if test_loader is not None:
```

Comment:
Q: don't we want to run eval on test at the end?

Reply:
I'm not sure if we want it to be a separate `evaluate` command.